PULL-REQUEST.md

```diff
@@ -0,0 +1,2 @@
+## Pull Request Template
+
```

Comment:
This is a sentence made up of letters.

Reply:
Correct! Forgot to add the content, I will add that now.